### PR TITLE
Move ManagedServiceIdentity to main

### PR DESCRIPTION
**Pull request by BethanyZhou**
## Description

Move https://github.com/Azure/azure-powershell/pull/17204 to main

## Checklist

- [ ] I have read the [_Submitting Changes_](../blob/master/CONTRIBUTING.md#submitting-changes) section of [`CONTRIBUTING.md`](../blob/master/CONTRIBUTING.md)
- [ ] The title of the PR is clear and informative
- [ ] The appropriate `ChangeLog.md` file(s) has been updated:
    - For any service, the `ChangeLog.md` file can be found at `src/{{SERVICE}}/{{SERVICE}}/ChangeLog.md`
    - A snippet outlining the change(s) made in the PR should be written under the `## Upcoming Release` header -- no new version header should be added
- [ ] The PR does not introduce [breaking changes](../blob/master/documentation/breaking-changes/breaking-changes-definition.md)
- [ ] If applicable, the changes made in the PR have proper test coverage
- [ ] For public API changes to cmdlets:
    - [ ] a cmdlet design review was approved for the changes in [this repository](https://github.com/Azure/azure-powershell-cmdlet-review-pr) (_Microsoft internal only_)
        - {Please put the link here}
    - [ ] the markdown help files have been regenerated using the commands listed [here](../blob/master/documentation/development-docs/help-generation.md#updating-all-markdown-files-in-a-module)

### CODE REVIEW

File: src/ManagedServiceIdentity/Properties/AssemblyInfo.cs [hunk 1]
```diff
@@ -0,0 +1,28 @@
...
+[assembly: AssemblyTitle("Microsoft Azure Powershell - ManagedServiceIdentity")]
+[assembly: AssemblyCompany(Microsoft.WindowsAzure.Commands.Common.AzurePowerShell.AssemblyCompany)]
+[assembly: AssemblyProduct(Microsoft.WindowsAzure.Commands.Common.AzurePowerShell.AssemblyProduct)]
+[assembly: AssemblyCopyright(Microsoft.WindowsAzure.Commands.Common.AzurePowerShell.AssemblyCopyright)]
+
+[assembly: ComVisible(false)]
+[assembly: CLSCompliant(false)]
+[assembly: Guid("1d0ed657-ce71-49b5-bf60-d2fbb7982d0e")]
+[assembly: AssemblyVersion("0.1.0")]
+[assembly: AssemblyFileVersion("0.1.0")]
```
Comment (BethanyZhou) [inline src/ManagedServiceIdentity/Properties/AssemblyInfo.cs:28]:
```suggestion
[assembly: AssemblyVersion("0.7.3")]
[assembly: AssemblyFileVersion("0.7.3")]
```

**Comment by isra-fel** [comment]
Static analysis seemed to have errors, could you check if it needs to be fixed? 

https://dev.azure.com/azure-sdk/public/_build/results?buildId=1442850&view=logs&j=e1b7b984-9f58-529f-7c5a-f15f8e35cfa6&t=e8689ae8-b216-55f7-b207-4831f84ff98a&l=819

**Comment by BethanyZhou** [comment]
Thanks @isra-fel .

**Review by isra-fel — APPROVED** [review]
LGTM

**Pull request merged**